bHY6yh-d11545/cod-vyg67d_bHY6yh-de11545.xml

```diff
@@ -0,0 +1,159 @@
+          </p>
+        </availability>
+      </publicationStmt>
+      <sourceDesc>
+        <listWit>
+          <witness xml:id="O" n="cod-vyg67d">Vatican, Biblioteca Apostolica Vaticana, Vat. lat. 8175</witness>
+        </listWit>
+      </sourceDesc>
+    </fileDesc>
+    <encodingDesc>
+      <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+      <editorialDecl>
+        <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+          guidelines for a diplomatic edition.</p>
+      </editorialDecl>
+    </encodingDesc>
+    <revisionDesc status="draft">
+      <listChange>
+        <change when="2021-05-18" status="draft" n="0.0.0">
+          <p>Created file for the first time.</p>
+        </change>
+      </listChange>
+    </revisionDesc>
+  </teiHeader>
+  <text xml:lang="la">
+    <front>
+      <div xml:id="starts-on">
+        <pb ed="#O" n="29-v"/>
+        <cb ed="#O" n="b"/>
+      </div>
+    </front>
+    <body>
+      <div xml:id="bHY6yh-d11545"><!-- p1l2-r2 -->
+        <head xml:id="bHY6yh-d11545-Hd1e108">Rubrica 2</head>
+        <head xml:id="bHY6yh-d11545-Hd1e111" type="question-title">De honesto et licito et inhonesto seu turpi</head>
+        <p xml:id="bHY6yh-d11545-d1e114">
+           <lb ed="#O"/>De honesto et licito et in honesto s
+           <lb ed="#O"/>eu turpi. Rubrica. 2.
+        </p>
+        <p xml:id="bHY6yh-d11545-d1e121">
+           <lb ed="#O"/><ref xml:id="bHY6yh-d11545-Rd1e118">ARistoteles rectoricorum libro 1. 
+           capitulo 13.</ref> ¶ <quote xml:id="bHY6yh-d11545-Qd1e129">honestum est quodcumque 
+           <lb ed="#O"/>propter se eligibile existens lauda<lb ed="#O" break="no"/>bile 
+           est aut quodcumque bonum existens 
+           <lb ed="#O"/>delectabile est quia bonum.</quote>
+        </p>
+        <p xml:id="bHY6yh-d11545-d1e137">
+           <!--line bumps down-->
+           <lb ed="#O"/><ref xml:id="bHY6yh-d11545-Rd1e141">TErentius in euinco. <!--line bumps up--> capitulo 23.</ref> ¶ <quote xml:id="bHY6yh-d11545-Qd1e141">Quis <!--line bumps back down--> homo pro 
+           <lb ed="#O"/>mecho unquam uidit in domo 
+           <lb ed="#O"/>meretricis prendi quemquam:</quote> ¶ <ref xml:id="bHY6yh-d11545-Rd1e150"><sic>Inea<lb ed="#O" break="no"/>ncontumerumeton</sic>
+           capitulo 12.</ref> ¶ <quote xml:id="bHY6yh-d11545-Qd1e155">Non
+           licet hominem esse ita ult si res 
+           <lb ed="#O"/>non sunt.</quote>
+           <pb ed="#O" n="30-r"/>
+           <cb ed="#O" n="a"/>
+        </p>
+        <p xml:id="bHY6yh-d11545-d1e152">
+           <ref xml:id="bHY6yh-d11545-Rd1e168">Tullius de fine bonorum et malo<lb ed="#O" break="no"/>rum 
+           libro 2.</ref> ¶ <quote xml:id="bHY6yh-d11545-Qd1e156">honestum intelligimus 
```

Comment:
if a entry starts on a new page or column, the break should be at the beginning of the paragraph, not at the end of the previous paragraph. And in this case I think you're also missing a line break.
```suggestion
           <lb ed="#O"/>non sunt.</quote>
        </p>
        <p xml:id="bHY6yh-d11545-d1e152">
           <pb ed="#O" n="30-r"/>
           <cb ed="#O" n="a"/>
           <lb ed="#O"/><ref xml:id="bHY6yh-d11545-Rd1e168">Tullius de fine bonorum et malo<lb ed="#O" break="no"/>rum 
           libro 2.</ref> ¶ <quote xml:id="bHY6yh-d11545-Qd1e156">honestum intelligimus 
```